netket/graph/lattice.py

```diff
@@ -246,6 +315,114 @@ def atoms_coord(self):
         """
         return self._atoms_coord
 
+    @property
+    def coords(self):
+        """
+        Returns list of coordinates of lattice points
+        """
+
+        return self._coords
+
+    def translation_perm(self):
+        perms = []
+        for vec in self.basis_vectors:
+            perm = []
+            for coord in self._coords:
+                hash_coord = coord.copy() + vec
+                hash_coord = _np.matmul(hash_coord, self._inv_dims) % 1
+                # make sure 1 and 0 are treated the same
```

Comment:
Great adding a comment, but i don't get it.
Can you explain better?

Reply:
I think you mean that you are trying to keep only the decimal part?

Reply:
This was a comment from a previous version that no longer makes sense.